tests/foreman/cli/test_repository.py

```diff
@@ -662,8 +695,10 @@ def test_negative_create_with_auth_url_with_special_characters(self):
         :CaseImportance: Critical
         """
         # get a list of valid credentials without quoting them
-        for cred in [creds for creds in valid_http_credentials()
-                     if creds['quote'] is True]:
+        for cred in [
+            creds for creds in valid_http_credentials()
+            if creds['quote'] is True
+        ]:
```

Comment:
this also can be updated to 
```python
for cred in [
    creds for creds in valid_http_credentials()
    if creds['quote'] is True
]:
    url_encoded = FAKE_5_YUM_REPO.format(cred['login'], cred['pass'])		                  
    with self.subTest(url_encoded):
        with self.assertRaises(CLIFactoryError):
        ...

```

Reply:
sure, but all these changes were not my initial intention anyway=)
so it can be fixed in next PRs

Reply:
yes I know, but one more test line and we finish with all this refactoring for ever (I hope) :)

Reply:
that is applicable for two tests, not one, but no problem
Fixed